DependencyInjection/FOSUserExtension.php

```diff
@@ -133,6 +132,22 @@ public function load(array $configs, ContainerBuilder $container)
         if (!empty($config['group'])) {
             $this->loadGroups($config['group'], $container, $loader, $config['db_driver']);
         }
+
+        if ($container->hasParameter('fos_user.registration.confirmation.from_email') || $container->hasParameter('fos_user.resetting.email.from_email')) {
+            $mailer = $config['service']['mailer'];
+            $container->setAlias('fos_user.mailer', $mailer);
+            $mailerDefinition = $container->getDefinition($mailer);
```

Comment:
If it is, then it should be specified in the `$config['service']['mailer']` config right ? So I guess the improvement must be done on that line instead ? https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2054/files#diff-1dc28cb73010a64f779af53832fdbefdR149